server/server.go

```diff
@@ -106,15 +103,23 @@ func (s *Server) Teardown() {
 // in order to provide a nicer API when setting up the server.
 func (s *Server) EnableTracing() {
 	grpc.EnableTracing = true
-	logger.Notice("Enabled gRPC tracing")
+	s.logger.Notice("Enabled gRPC tracing")
+}
+
+func (s *Server) SetLogLevel(level string) error {
+	return s.logger.SetLevel(level)
+}
+
+func (s *Server) DisableLogging() {
```

Comment:
Same here, please remove this function and disable logging on the logger prior assigning it to the struct instead.

Reply:
In addition, I am removing `func Disable()` on the `Logger` interface in favor of  newly introduced `struct NullLogger` that just discards all the logging messages.